.gitignore

```diff
@@ -70,6 +70,9 @@ typings/
 # parcel-bundler cache (https://parceljs.org/)
 .cache
 
+# IntelliJ config dir
+.idea/
```

Comment:
Can you regenerate this gitignore using the gitignore.io link at the top and add vim and IntelliJ?